src/components/icon/icon.jsx

```diff
@@ -6,14 +6,18 @@ import styles from './icon.styles.pcss';
 import IMAGES from './icons';
```

Comment:
add hover and disabled state, also pls check that everything looks like on design